qir/qat/ValidationPass/ValidationPassConfiguration.cpp

```diff
@@ -67,9 +67,9 @@ namespace quantum
             profile.allow_internal_calls_     = false;
             profile.allowlist_external_calls_ = true;
             profile.allowlist_opcodes_        = true;
-            profile.opcodes_                  = Set{"ret",  "inttoptr", "br",   "phi", "add", "sub", "mul",  "fadd",
-                                   "fsub", "fmul",     "ashr", "and", "or",  "xor", "icmp", "fcmp"};
-            profile.external_calls_           = Set{
+            profile.opcodes_        = Set{"call", "ret",  "inttoptr", "br",  "phi", "add", "sub",  "mul", "fadd",
+                                   "fsub", "fmul", "ashr",     "and", "or",  "xor", "icmp", "fcmp"};
```

Comment:
nit: spacing before the `"and"`

Reply:
Turns out that clang-format enforces this... without the spacing it will fail CI

Reply:
That seems like an unexpected clang-format enforcement :P